q2_types/feature_table/_transformer.py

```diff
@@ -12,26 +12,36 @@
 import pandas as pd
 import qiime
 
-from . import BIOMV1Format
+from . import BIOMV1Format, BIOMV210Format
 from ..plugin_setup import plugin
 
 
+def _get_generated_by():
+    return 'qiime %s' % qiime.__version__
+
+
 @plugin.register_transformer
 def _1(data: biom.Table) -> BIOMV1Format:
+    # TODO: issue a warning about limited support for this format
```

Comment:
I don't think this needs to hold anything up (its just a TODO comment at this point), I was just asking a bigger-picture type question.